drive/utils.bal

```diff
@@ -662,3 +665,21 @@ isolated function uploadFileUsingByteArray(http:Client httpClient, byte[] byteAr
     }
 }
 
+# Utility function to retrieve File content as a binary content with its MIME type.
+# 
+# + httpClient - Drive client
+# + path - GET URI path
+# + return - `FileContent` if successful, `error` if not.
+isolated function generateRecordFileContent(http:Client httpClient, string path) returns @tainted FileContent|error {
+    http:Response response = <http:Response> check httpClient->get(<@untainted>path);
+    if (response.statusCode is http:STATUS_OK) {
+        FileContent fileContent  = {
+            content: check response.getBinaryPayload(),
+            mimeType: response.getContentType()
+        }; 
+        return fileContent;
+    } else {
+        return error(ERR_EXTRACTING_BINARY_PAYLOAD);
```

Comment:
Just a suggestion, We can use `fail error(<Error Messsage>)`;

Reply:
Yeah. That is better. I changed it. /Thanks